### ISSUE TITLE
[Bug] Reading a default duration of 0 from the config file will incorrectly display as 10 sec upon restarting

### ISSUE THREAD

**Issue by snailfart** [lxqt/lxqt-notificationd#211]
If the previous value is 0, selecting "Advanced Settings" in lxqt-config-notificationd always displays the predefined duration of 10 sec.
This issue is however only cosmetical;  it doesn't reset the timeout I set previously until alter it manually.

lxqt-notificationd Version: 0.15.0

**Comment by zhrexl** [comment]
I agree that its not a good approach. I believe the idea was to prevent 0 sec. but in that case it should save the 10 sec. timeout. 

The thing is, if you don't want to see notifications from a specific app you should blacklist it and if you don't want to show any notification then a notification-daemon should not be running but thats not a good option (some apps will not work if there isnt an notification-daemon running, i.e. electron apps) perhaps implementing a "silence mode" would be a good idea?

In that case the daemon would receive the notification but not show it, the same as setting timeout to 0.

**Comment by snailfart** [comment]
I understand but this was not my intention at all, I wanted the notifications to _never_ fade until I click them away, which worked very well, I can even set it to 0 through the GUI, it will just return to 10 sec after restarting the application.

**Comment by zhrexl** [comment]
Oh, I see. I misunderstood what you wanted to achieve.
I'll take a look to see what can be improved.
